Makefile

```diff
@@ -35,8 +35,8 @@ generate:
 apidocs:
 	hack/dockerized "./hack/generate.sh && ./hack/gen-swagger-doc/gen-swagger-docs.sh v1 html"
 
-client-python:
-	hack/dockerized "./hack/generate.sh && TRAVIS_TAG=${TRAVIS_TAG} ./hack/gen-client-python/generate.sh"
+client-python: generate
```

Comment:
This is not the right direction. Intentionally the logic should be in the bash scripts to avoid starting and stopping the containers multiple times because for many engineers it makes the difference between fast and seamless execution and having to wait for 5-10 seconds longer. Not so important for this target, but important in general. We can fix it in another PR.